# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/out/gen/angle/id/commit.h
# Commit: 302c30b598b0
# Full Hash: 302c30b598b0ed12b212c19f86256723dd362835
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-07 04:06:39
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r 6e2d679c7168 -r 302c30b598b0 gfx/angle/checkout/out/gen/angle/id/commit.h
--- a/gfx/angle/checkout/out/gen/angle/id/commit.h	Wed May 06 21:21:46 2020 +0000
+++ b/gfx/angle/checkout/out/gen/angle/id/commit.h	Wed May 06 22:04:28 2020 +0000
@@ -1,3 +1,3 @@
-#define ANGLE_COMMIT_HASH "3885ee272685"
+#define ANGLE_COMMIT_HASH "8df54289d717"
 #define ANGLE_COMMIT_HASH_SIZE 12
-#define ANGLE_COMMIT_DATE "2020-01-22 18:48:23 -0800"
+#define ANGLE_COMMIT_DATE "2020-05-06 11:04:42 -0700"